src/inc_xram.a51

```diff
@@ -0,0 +1,80 @@
+;    Copyright (c) 2007, 2008 Florian MAZEN and Pierre COL
+;    
+;    This program is free software: you can redistribute it and/or modify
+;    it under the terms of the GNU General Public License as published by
+;    the Free Software Foundation, either version 3 of the License, or
+;    (at your option) any later version.
+;
+;    This program is distributed in the hope that it will be useful,
+;    but WITHOUT ANY WARRANTY; without even the implied warranty of
+;    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;    GNU General Public License for more details.
+;
+;    You should have received a copy of the GNU General Public License
+;    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+; Segmentation of external memory now moved to an own file so memory structure can be seen easily
+; and DPTR can be loaded with 16 Bit values, also
+
+
+XSEG	AT 0000H						; RAM_Area_Config, don't change the order of variables					
+
+RAM_ID_CODE:		DS	1
+RAM_CONFIG_SUM:		DS	1
```

Comment:
Yes, I found this feature. I can not adjust the number of spaces, but I counted the editor is inserting 4 spaces, now.
I fear that I already inserted lots of "tabs" instead of spaces, how to deal with this ?

Sorry, seems I provide you lots of work, that was not my intention

Reply:
Don't worry. That depends of you editor, most of them will automatically do the conversion on save